packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -203,10 +241,15 @@ export function drawLivenessOvalInCanvas({
   const ctx = canvas.getContext('2d');
 
   if (ctx) {
+    ctx.restore();
     ctx.clearRect(0, 0, canvasWidth, canvasHeight);
 
     // fill the canvas with a transparent rectangle
-    ctx.fillStyle = 'rgba(255, 255, 255, 1.0)';
+    ctx.fillStyle = getComputedStyle(canvas).getPropertyValue(
+      isStartScreen
+        ? '--amplify-colors-background-primary'
+        : '--amplify-colors-white'
```

Comment:
These are the same by default, what is this logic doing?

Reply:
The way rekognition wanted it to be was that on the start screen we would respect the theme background color.

When we are in the check we would always use white to ensure that we have a higher chance of passing the freshness color check. This logic makes it so that when changing to dark mode we ensure that it is always white on the check screen.

Reply:
Should we call this out in documentation, so customers aren't confused by this behavior?

Reply:
If we want it to always be white should we just hard-code it to be `#fff` ? Because technically, customers can override the color-white token to be any color.

Reply:
I can update that. Will also update the color for the cancel button to be hard coded as well since they always want that to stay the same color